modules/logging-loki-cli-install.adoc

```diff
@@ -37,9 +69,186 @@ spec:
 <2> Specify `stable`, or `stable-5.<y>` as the channel.
 <3> Specify `redhat-operators`. If your {product-title} cluster is installed on a restricted network, also known as a disconnected cluster, specify the name of the `CatalogSource` object you created when you configured the Operator Lifecycle Manager (OLM).
 
-. Apply the `Subscription` object:
+
+. Apply the namespace object YAML configuration that follows:
++
+[source,terminal]
+----
+$ oc apply -f openshift_logging_namespace.yaml
+----
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+name: openshift-logging # <1>
+  annotations:
+    openshift.io/node-selector: ""
+  labels:
+    openshift.io/cluster-logging: "true"
+    openshift.io/cluster-monitoring: "true" # <2>
+----
+<1> The Red Hat OpenShift Logging Operator is only deployable to the `openshift-logging` namespace.
+<2> Enables metric service discovery.
+
+. Apply the Operator Group object YAML configuration that follows:
++
+[source,terminal]
+----
+$ oc apply -f olo-og.yaml.yaml
+----
++
+--
```

Comment:
These -- lines are messing something up in the preview. It's not aligned with Step 3, then causes numbering to start over.